libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/DialogFragment.kt

```diff
@@ -92,11 +96,31 @@ class DialogFragment internal constructor() : BaseBottomSheetDialog() {
                     dialogListViewModel.clearSearch()
                 }
 
-                constraintLayout.descendantFocusability = FOCUS_AFTER_DESCENDANTS
-
                 setUpViewModel(items)
             }
         }
+
```

Comment:
We keep the block in the comments. Is it intentional 🤔

Reply:
Nope and it is deleted :)